finetune/utils.py

```diff
@@ -130,40 +86,17 @@ def soft_split(*xs, n_splits=None):
         start = tf.minimum(i * n_per, current_batch_size)
         end = tf.minimum((i + 1) * n_per, current_batch_size)
         i_range = tf.range(start, end)
-        yield [tf.gather(x, i_range) for x in xs]
+        yield [tf.gather(x, i_range) if x is not None else None for x in xs]
```

Comment:
I think we can cut the soft_split function if we get the distribution strategy working, right?

Reply:
got em